chemie/elections/models.py

```diff
@@ -1,51 +1,184 @@
-from django.db import models
+from django.db import models, transaction
+from django.utils.crypto import get_random_string
 from django.contrib.auth.models import User
-from django.db import transaction
+import uuid
+import datetime
+from django.shortcuts import render_to_response, get_object_or_404, render
+
+
+
+"""
+POTITION_TYPE_CHOICES= (
```

Comment:
Lite dynamisk, burde være en egen modell. Har skrevet dette på PRen også.

Reply:
This could be removed, since the Position model replaces it.